utFileMonitor.py

```diff
@@ -0,0 +1,119 @@
+import sys
```

Comment:
- design requirement first
- hold meeting to dsicuss design
- sign off design.
- implement based on decision.

Reply:
- Goal: For the class

- Monitor for events coming into a file on the target.
- Wait for an event to arrive, then inform the caller.
- Set up an option start event marker, and end event marker.
- if the full message is received within the timeout, then the callback is triggered
- if the message timeout, then the callback is not triggered.
- Each instance of the class will create a monitoring thread
  - In the case where multiple sessions are required, the session class must support multiple instance.